<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration</class>
 <widget class="QDialog" name="registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="registration_groupbox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>431</width>
     <height>631</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(225, 225, 225);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="pass_labelr">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass_register">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="registerBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 170, 170);</string>
    </property>
    <property name="text">
     <string>Submit !</string>
    </property>
   </widget>
   <widget class="QLabel" name="email_labelr">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_register">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="fname_register">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fname_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Firstname</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lname_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Lastname</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lname_register">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone_register">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="phone_labelr">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_register">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="adress_labelr">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="fb_link_labelr">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>FB link</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fb_link_register">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pass_labelr</zorder>
   <zorder>pass_register</zorder>
   <zorder>email_labelr</zorder>
   <zorder>email_register</zorder>
   <zorder>registerBtn</zorder>
   <zorder>fname_register</zorder>
   <zorder>fname_label</zorder>
   <zorder>lname_label</zorder>
   <zorder>lname_register</zorder>
   <zorder>phone_register</zorder>
   <zorder>phone_labelr</zorder>
   <zorder>address_register</zorder>
   <zorder>adress_labelr</zorder>
   <zorder>fb_link_labelr</zorder>
   <zorder>fb_link_register</zorder>
  </widget>
  <widget class="QPushButton" name="sign_inWinBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 170);</string>
   </property>
   <property name="text">
    <string>Sign In</string>
   </property>
  </widget>
  <widget class="QLabel" name="reg_right_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
